jitouch/Jitouch/Gesture.m

```diff
@@ -2958,7 +2958,6 @@ - (id)init {
                     MTRegisterContactFrameCallback(device, trackpadCallback);
```

Comment:
should also add 
`familyID == 113 // m2 mbp with touch bar`
to the if statement above, after `familyID == 105 // macbook with touch bar`, since we know that 113 is associated with a device

Reply:
done